vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -126,6 +127,17 @@ void AdsImpl::InitializeStep4(const Result result) {
 
   NotificationAllowedCheck(false);
 
+  //ads notifications don't sustain reboot, so remove all
+  auto ads_shown_history = client_->GetAdsShownHistory();
+  if (!ads_shown_history.empty()) {
+    uint64_t ad_shown_timestamp = ads_shown_history.front();
```

Comment:
`ad_shown_timestamp` is in `base::Time` `ToDoubleT` format. Should `boot_timestamp` also use `ToDoubleT` instead of `InSeconds`?

Reply:
`ad_shown_timestamp` has `uint64_t` type.
See signature below:
`std::deque<uint64_t> Client::GetAdsShownHistory()`